opengraph/dom-test/conftest.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import pytest
+from bs4 import BeautifulSoup
+
+
+# Load url and parse it
+@pytest.fixture
+def meta(response_obj):
+    soup = BeautifulSoup(response_obj.content, "html.parser")
+    return soup.find_all("meta")
+
+
+@pytest.fixture
+def response_obj():
+    return requests.get('http://nightly.mozilla.org')
```

Comment:
something like:

~~~
for url in config_urls:
    url = 'https://{0}'.format(url)
~~~